includes/key_listener.h

```diff
@@ -0,0 +1,25 @@
+#ifndef KEY_LISTENER_H
+# define KEY_LISTENER_H
+
+# include "libft.h"
+
```

Comment:
Magic numbers! Maybe put a comment with where you got these numbers, since I have no clue right now